cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -872,13 +911,14 @@ func leaderElectAndRun(ctx context.Context, c *config.CompletedConfig, lockIdent
 	}
 
 	leaderelection.RunOrDie(ctx, leaderelection.LeaderElectionConfig{
-		Lock:          rl,
-		LeaseDuration: c.ComponentConfig.Generic.LeaderElection.LeaseDuration.Duration,
-		RenewDeadline: c.ComponentConfig.Generic.LeaderElection.RenewDeadline.Duration,
-		RetryPeriod:   c.ComponentConfig.Generic.LeaderElection.RetryPeriod.Duration,
-		Callbacks:     callbacks,
-		WatchDog:      electionChecker,
-		Name:          leaseName,
+		Lock:                      rl,
+		LeaseDuration:             c.ComponentConfig.Generic.LeaderElection.LeaseDuration.Duration,
+		RenewDeadline:             c.ComponentConfig.Generic.LeaderElection.RenewDeadline.Duration,
+		RetryPeriod:               c.ComponentConfig.Generic.LeaderElection.RetryPeriod.Duration,
+		Callbacks:                 callbacks,
+		WatchDog:                  electionChecker,
+		Name:                      leaseName,
+		CoordinatedLeaderElection: utilfeature.DefaultFeatureGate.Enabled(kubefeatures.CoordinatedLeaderElection),
```

Comment:
```suggestion
		Coordinated: utilfeature.DefaultFeatureGate.Enabled(kubefeatures.CoordinatedLeaderElection),
```
The context is clear. No need for the suffix.